vrs/marten-construction-script.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Daniel Marten 01-04-2022
+# Constructing a ~100k Variant Test Database for GA4GH-VRS Annotation Script
+# NOTE: adapted from a Jupyter Notebook, so excuse any formatting quirks! 
+
+import hail as hl
+from random import sample
+
+# importing the gnomAD v3.1.2 full variant table, approx 750,000,000 variants 
+
+ht_whole = hl.read_table('gs://gcp-public-data--gnomad/release/3.1.2/ht/genomes/gnomad.genomes.v3.1.2.sites.ht',_n_partitions=10000)
+
+
+# PREVIOUSLY CALCULATED FREQUENCY STATISTICS: 
+# frequency of the following in the general v3 database , based off of frequencties in random 10k and 100k databases
+
+indel_freq = 0.1460 # based on the amount of non-SNV variants in a 10k and 100k dataset
+ref_freq = 0.3704 # fraction of Long References (>3 bp) in a 10k and 100k dataset
+var_freq = 0.4650 # fraction of Long Variants (>3 bp) in a 10k and 100k dataset
+x_freq = 0.04004 # ^^ for X-Chromosome Variants
+y_freq = 0.001531 # ^^ for Y-Chromosome Variants
+
+# Constructing tables of samples from this 
+# Makeup: 50k random, 10k extra indels, 10k more long references, 10k long variants, 10k sex chromosome
+# --> (9k X, 1k Y)
+# + 10k (actual results are 20k) multiallelic variants 
+
+finalSampleFreq = 50000/759000000 # the sampling rate to get 50000 variants of the 759,000,000 read in
```

Comment:
in general, it's better to set variables rather than hardcoding numbers. this could be something like
```
total_variants = ht_whole.count()
finalSampleFreq = n_var/total_variants
```
and n_var could be set by a constant or on runtime using script arguments